src/diffusers/loaders/lora_pipeline.py

```diff
@@ -1489,10 +1489,10 @@ class FluxLoraLoaderMixin(LoraBaseMixin):
 
     @classmethod
     @validate_hf_hub_args
-    # Copied from diffusers.loaders.lora_pipeline.SD3LoraLoaderMixin.lora_state_dict
     def lora_state_dict(
         cls,
         pretrained_model_name_or_path_or_dict: Union[str, Dict[str, torch.Tensor]],
+        return_alphas: bool = False,
```

Comment:
is this argument there to prevent breaking? i.e. `load_state_dict()` would be used outside our `load_lora_weights` method?

Reply:
yes

Reply:
Yes. We use `lora_state_dict()` in the training (example usage outside of `load_lora_weights()`).